IdentityServer/IdentityServer/Config.cs

```diff
@@ -50,19 +50,6 @@ public static class Config
                     AllowedScopes = { "News.API" }
                 },
                 new Client
-                {
-                    ClientId = "client_id",
```

Comment:
Why do you delete client with client credentials grant type?

Reply:
this is duplicated code, after conflict resolution